vm/transaction.go

```diff
@@ -17,6 +17,7 @@ func marshalTxn(txn core.Transaction) (json.RawMessage, error) {
 	t := adaptTransaction(txn)
 	version := (*core.TransactionVersion)(t.Version)
 	txnAndQueryBit := struct {
+		// what is query bit?
```

Comment:
it is for txns sent to a full node for fee estimates and stuff
So, users still sign the txn but with a query bit
Which invalidates the txn in sequencing context
So someone can't just use a txn that user signed for a RPC request.